### PR TITLE
Show proper error message if DbContext could not edited. Fixes #140

### LINKED ISSUE #140: Error occurs using classes from another class project while scaffolding with rc1-final
When I try to add a new scaffolded item from context menu inside Visual Studio Community Edition, I get an error saying "There was an error running the selected code generator: 'Value cannot be null. Parameter name: oldTree'".

I am trying to get model and context information from another class project inside the same solution.
addcontroller
addcontrollererror

### PR DESCRIPTION
**Pull request by prafullbhosale**
cc @abpiskunov

### CODE REVIEW

File: src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/MessageStrings.resx [hunk 1]
```diff
@@ -135,6 +135,9 @@
   <data name="FailedToEditStartup" xml:space="preserve">
     <value>Scaffolding failed to edit Startup class to register the new Context using Dependency Injection.</value>
   </data>
+  <data name="ModelTypeCouldNotBeAdded" xml:space="preserve">
+    <value>Could not add Model type '{0}' to DbContext '{1}'. If '{1}' is in a dependency, make sure it has a DbSet property for '{0}'.</value>
```
Comment (abpiskunov) [inline src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/MessageStrings.resx]: 'If {1} is in a dependency'  - sounds unclear , could we rephrase it?

Comment (prafullbhosale) [inline src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/MessageStrings.resx]: I agree. Would it make sense to just say:
`Please make sure that '{1}' has a DbSet property for '{0}'.` ?

Comment (abpiskunov) [inline src/Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore/MessageStrings.resx]: yes

**Comment by abpiskunov** [comment]
:shipit:

**Pull request merged**